<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientClass</class>
 <widget class="QWidget" name="ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinChat</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="layout_message">
     <item>
      <widget class="QTextBrowser" name="textBrowser_Display"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_set">
       <item>
        <widget class="QFontComboBox" name="font_changeFont">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="button_textBold">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="button_textItalic">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="button_textULine">
         <property name="text">
          <string>U</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="button_textColor">
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_Send">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_control">
       <item>
        <widget class="QLabel" name="label_AD">
         <property name="text">
          <string>Simon:广告位招租。^-^</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Button_Close">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Send">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="layout_info">
     <item row="0" column="0">
      <widget class="QTableWidget" name="table_users">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>UserName</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ipAddress</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lable_users">
       <property name="text">
        <string>在线人数：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
